packages/orbit-components/src/Accordion/AccordionSection/components/SectionHeader.tsx

```diff
@@ -21,6 +23,9 @@ const AccordionSectionHeader = ({
   onExpand,
   expandable,
   dataTest,
+  ariaControls,
+  id,
+  ariaLabelledby,
```

Comment:
Just wondering… why do we need this? What are we trying to label exactly?

Reply:
We don't need this anymore, that's leftover from the previous implementation. I also removed `id` as it seems it's not needed anymore either.